embd.go

```diff
@@ -176,12 +194,12 @@ package {{.Pkg}}
 {{range .Files}}
 // {{.VarName}} contains contents of "{{.Path}}" file.
 var {{.VarName}} = []byte("{{range .DataFragments}}{{.}}{{end}}")
-{{end}}`[1:]
+{{end}}
 
-/*
-var dirTemplate = `
-var {NAME} = map[string][]byte{
-{ENTRIES}
+{{range $index, $element := .Dirs}}
+var Dir_{{$index}} = map[string][]byte{
```

Comment:
Ok; but please:
1. keep the template as much gofmt-ed as possible anyway (as I asked for earlier);
2. rename 'src' to 'code'.